src/app/services/admin.service.ts

```diff
@@ -25,4 +25,17 @@ export class AdminService {
       id
     );
   }
+
+  createBorrower(body) {
+    return this.http.post(
+      environment.api_endpoint +
+      environment.admin_endpoint +
+      environment.createBorrower + "?name=" +
+      body.name + "&address=" + body.address +
```

Comment:
What if the user puts a `&` into the address? Pass the user's data as the POST body, not in the URL.

Reply:
But the server only accepts posts in the url. I do not think it takes anything in the body

Reply:
Spring accepts parameters either in the URL or in the body, but remember that one of the main "selling points" of POST as opposed to GET is that it can *have* a body and can pass an unlimited amount of data via that body, while with GET the URL is of limited size. You can look at what I did in my Author implementation for a model.